dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace CopyObject
+{
+    public class CopyObject
+    {
+        // Specify your bucket region (an example region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the names for the Simple Storage Service objects
+        private const string SOURCE_BUCKET_NAME = "*** provide the name of the bucket with source object ***";
+        private const string DESTINATION_BUCKET_NAME = "*** provide the name of the bucket to copy the object to ***";
+        private const string SOURCE_OBJ_KEY = "*** provide the name of object to copy ***";
+        private const string DESTINATION_OBJ_KEY = "*** provide the destination object key name ***";
+
+        static void Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+
+            Console.WriteLine($"Copying {SOURCE_OBJ_KEY} from {SOURCE_BUCKET_NAME} to ");
+            Console.WriteLine($"{DESTINATION_BUCKET_NAME} as {DESTINATION_OBJ_KEY}");
+
+            var response = CopyingObjectAsync(
```

Comment:
Looks like "response" isn't actually used and therefore can probably be safely removed?

Reply:
It is now. I've extracted the actual result from the method return and used it to check for successful completion.